postgre-client/src/asset_filter_client.rs

```diff
@@ -172,13 +172,13 @@ fn add_filter_clause<'a>(
         group_clause_required = true;
     }
     if let Some(ref token_type) = filter.token_type {
-        if token_type == &TokenType::All {
+        if token_type == &TokenType::All && filter.owner_address.is_some() {
```

Comment:
Are we sure it will not break any `serch` queries? Because this function is used by that enpoint as well.
For example if someone will call `serch` and put `TokenType::Fungible` but will not point owner, what gonna happen?

Reply:
There validation check in API that does not allow setting tokenType without setting ownerAddress